ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -36,6 +36,10 @@
   private static final int DEFAULT_WORLD_STATE_MAX_REQUESTS_WITHOUT_PROGRESS = 1000;
   private static final long DEFAULT_WORLD_STATE_MIN_MILLIS_BEFORE_STALLING =
       TimeUnit.MINUTES.toMillis(5);
+  private static final int DEFAULT_MAX_GET_BLOCK_HEADERS = 192;
+  private static final int DEFAULT_MAX_GET_BLOCK_BODIES = 128;
+  private static final int DEFAULT_MAX_GET_RECEIPTS = 256;
+  private static final int DEFAULT_MAX_GET_NODE_DATA = 384;
```

Comment:
These constants aren't really related to the synchronizer, I'd move them to `EthServer` or `EthProtocolManager`

Reply:
fixed